ote/src/cljs/ote/app/controller/operator_users.cljs

```diff
@@ -54,32 +54,68 @@
       (assoc-in [:manage-access :new-member-email] "")
       (assoc-in [:manage-access :new-member-loading?] false)))
 
-(define-event PostNewUser [email operator-id]
+(define-event PostNewUser [email ckan-group-id]
   {}
   (comm/post!
-    (str "transport-operator/" (url-util/encode-url-component operator-id) "/users")
+    (str "transport-operator/" (url-util/encode-url-component ckan-group-id) "/users")
     {:email email}
     {:on-success (tuck/send-async! ->NewUserSuccess)
      :on-failure (tuck/send-async! ->NewUserFailure)})
   (assoc-in app [:manage-access :new-member-loading?] true))
 
-;; TODO: remove removed user from app-state
-(define-event RemoveMemberSuccess [result]
+(define-event RemoveMemberSuccess [result member]
   {}
-  app)
+  (let [users (get-in app [:manage-access :users])
+        new-users (filterv
+                    #(not= (:id member) (:id %))
+                    users)]
+    (-> app
+      (assoc-in [:manage-access :confirmation :open?] false)
+      (assoc-in [:manage-access :users] new-users))))
+
+(define-event RemoveTokenSuccess [result token]
```

Comment:
Almost copy paste of RemoveMemberSuccess fn. Consider creating one function with changing parameter